apps/mobile/src/components/Post/PostInput.tsx

```diff
@@ -34,67 +38,72 @@ export function PostInput({ value, onChange, tokenRef, mentions, onSelectionChan
 
   const { colorScheme } = useColorScheme();
 
-  const isTextTooLong = value.length >= MAX_LENGTH;
-  const characterCount = value.length;
+  const isTextTooLong = value.length >= MAX_POST_LENGTH;
 
   const inputPlaceHolder = useMemo(() => {
     return `Say something about "${token.definition?.name ?? 'this item'}"`;
   }, [token.definition?.name]);
 
   return (
-    <View
-      className="relative border bg-faint dark:bg-black-900 border-porcelain dark:border-black-500 pb-10"
-      style={{
-        height: 117,
-      }}
-    >
-      <MentionTextInput
-        value={value}
-        onChangeText={onChange}
-        className="px-3 pt-3 text-sm"
-        selectionColor={colorScheme === 'dark' ? colors.white : colors.black['800']}
-        placeholderTextColor={colorScheme === 'dark' ? colors.metal : colors.shadow}
-        multiline
-        onSelectionChange={(e: NativeSyntheticEvent<TextInputSelectionChangeEventData>) => {
-          onSelectionChange(e.nativeEvent.selection);
-        }}
-        maxLength={MAX_LENGTH}
-        autoCapitalize="none"
-        autoComplete="off"
-        keyboardAppearance={colorScheme}
-        placeholder={inputPlaceHolder}
+    <View className="space-y-2">
+      <View
+        className="relative border bg-faint dark:bg-black-900 border-porcelain dark:border-black-500"
         style={{
-          flex: 1,
-          color: colorScheme === 'dark' ? colors.white : colors.black['800'],
+          minHeight: 117,
+          height: 'auto',
         }}
-        mentions={mentions}
-      />
-
-      {isTextTooLong && (
-        <Typography
-          className="text-sm text-red absolute bottom-2 left-2"
-          font={{
-            family: 'ABCDiatype',
-            weight: 'Medium',
+      >
+        <MentionTextInput
+          value={value}
+          onChangeText={onChange}
+          className="p-3 text-sm"
+          selectionColor={colorScheme === 'dark' ? colors.white : colors.black['800']}
+          placeholderTextColor={colorScheme === 'dark' ? colors.metal : colors.shadow}
+          multiline
+          onSelectionChange={(e: NativeSyntheticEvent<TextInputSelectionChangeEventData>) => {
+            onSelectionChange(e.nativeEvent.selection);
```

Comment:
let's make sure to wrap in `useCallback`

Reply:
Updated!